src/test/java/com/intive/patronage22/szczecin/retroboard/service/BoardServiceTest.java

```diff
@@ -381,7 +379,8 @@ void patchBoardShouldUpdateNumberOfVotesWithoutProvidingName() {
         final var uid = "uid101";
         final var email = "username@test.pl";
         final var user = new User(uid, email, "displayName", Set.of());
-        final var board = buildBoard(user);
+        final var state = EnumStateDto.CREATED;
```

Comment:
you don't need to create state as separate variable if it's used only in line below
```suggestion
        final var board = buildBoard(user, EnumStateDto.CREATED);
```

Reply:
done